src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/ReadyToRunCodegenCompilationBuilder.cs

```diff
@@ -82,6 +82,16 @@ public override ICompilation ToCompilation()
             ModuleTokenResolver moduleTokenResolver = new ModuleTokenResolver(_compilationGroup, _context);
             SignatureContext signatureContext = new SignatureContext(_inputModule, moduleTokenResolver);
             CopiedCorHeaderNode corHeaderNode = new CopiedCorHeaderNode(_inputModule);
+            AttributePresenceFilterNode attributePresenceFilterNode = null;
+            
+            // Core library attributes are checked FAR more often than other dlls
+            // attributes, so produce a highly efficient table for determining if they are
+            // present. Other assemblies *MAY* benefit from this feature, but it doesn't show
+            // as useful at this time.
+            if (this._inputModule == this._inputModule.Context.SystemModule)
```

Comment:
We don't prefix member access with `this` per CoreFX coding guidelines.

Reply:
Fixed in next iteration

Reply:
Fixed